### PR TITLE
added option `--rocksdb.enforce-block-cache-size-limit`

### PR DESCRIPTION
**Pull request by jsteemann**
changed default value for `--rocksdb.block-cache-size`

### CODE REVIEW

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2278/

File: Documentation/Books/Manual/Programs/Arangod/Rocksdb.md [hunk 1]
```diff
@@ -38,7 +38,10 @@ Default: 2.
 The total amount of data to build up in all in-memory buffers (backed by log
 files). This option, together with the block cache size configuration option,
 can be used to limit memory usage. If set to 0, the memory usage is not limited.
-Default: 0 (disabled).
+
+If set to 0, this will cap memory usage for write buffers but may have an effect
```
Comment (graetzer) [inline Documentation/Books/Manual/Programs/Arangod/Rocksdb.md]: the documentation sounds wrong here

Comment (graetzer) [reply Documentation/Books/Manual/Programs/Arangod/Rocksdb.md]: 0 is unlimited ?

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2288/

**Pull request merged**